src/coreclr/jit/llvmcodegen.cpp

```diff
@@ -2091,7 +2089,7 @@ void Llvm::buildJTrue(GenTree* node)
     assert(condValue->getType() == Type::getInt1Ty(m_context->Context)); // Only relops expected.
 
     BasicBlock* srcBlock = CurrentBlock();
-    llvm::BasicBlock* jmpLlvmBlock = getFirstLlvmBlockForBlock(srcBlock->GetJumpDest());
+    llvm::BasicBlock* jmpLlvmBlock = getFirstLlvmBlockForBlock(srcBlock->GetTrueTarget());
     llvm::BasicBlock* nextLlvmBlock = getFirstLlvmBlockForBlock(srcBlock->Next());
```

Comment:
Optional: rename `jmpLlvmBlock`/`nextLlvmBlock` to `trueLlvmBlock`/`falseLlvmBlock`.
```suggestion
    llvm::BasicBlock* nextLlvmBlock = getFirstLlvmBlockForBlock(srcBlock->GetFalseTarget());
```